pkg/kritis/cryptolib/verify_detached.go

```diff
@@ -77,47 +67,36 @@ func verifyDetached(signature []byte, publicKey []byte, signingAlg SignatureAlgo
 
 	switch signingAlg {
 	case RsaSignPkcs12048Sha256, RsaSignPkcs13072Sha256, RsaSignPkcs14096Sha256, RsaSignPkcs14096Sha512:
-		var rsa_key *rsa.PublicKey
-		var ok bool
-		if rsa_key, ok = pub.(*rsa.PublicKey); !ok {
+		rsaKey, ok := pub.(*rsa.PublicKey)
+		if !ok {
 			return errors.New("expected rsa key")
 		}
-		hash, hashErr := getHash(signingAlg)
-		if hashErr != nil {
-			return errors.Wrap(hashErr, "error getting hash")
-		}
-		hashedPayload, payloadErr := hashPayload(payload, signingAlg)
-		if payloadErr != nil {
-			return errors.Wrap(payloadErr, "error hashing payload")
+		hash, hashedPayload, err := hashPayload(payload, signingAlg)
+		if err != nil {
+			return errors.Wrap(err, "error hashing payload")
```

Comment:
Optional: I would just return 'err' here because the caller of verifyDetached() is presumably going to errors.Wrap().

Reply:
makes sense. done.